group.yml

```diff
@@ -92,6 +92,10 @@ rhcos:
   require_consistency:
     driver-toolkit:
     - kernel  # since 9.3, kernel-rt is merged into the kernel package.
+    ose-ovn-kubernetes:
+    - libreswan
+    ovn-kubernetes-microshift:
```

Comment:
we don't support ipsec (libreswan) with microshift (yet). this is not needed at the moment.